app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -190,27 +186,47 @@ private fun HelpCenterHomeScreen(
                         is HelpCenterUiState.QuickLinkType.CommonClaimType -> onNavigateToCommonClaim(
                           quickLink.commonClaim,
                         )
+
                         is HelpCenterUiState.QuickLinkType.QuickActionType -> onQuickActionsSelected(
                           quickLink.quickAction,
                         )
                       }
                     },
-                    modifier = Modifier.fillMaxWidth(),
+                    modifier = Modifier
+                      .fillMaxWidth()
+                      .padding(horizontal = 16.dp),
                   ) {
                     Column(
                       verticalArrangement = Arrangement.Center,
-                      horizontalAlignment = Alignment.CenterHorizontally,
                     ) {
+                      Spacer(modifier = Modifier.height(12.dp))
                       Text(
                         text = when (quickLink) {
                           is HelpCenterUiState.QuickLinkType.CommonClaimType -> quickLink.commonClaim.title
                           is HelpCenterUiState.QuickLinkType.QuickActionType -> stringResource(
                             quickLink.quickAction.titleRes,
                           )
                         },
-                        textAlign = TextAlign.Center,
-                        modifier = Modifier.padding(16.dp),
+                        textAlign = TextAlign.Start,
+                        modifier = Modifier.padding(horizontal = 16.dp),
+                      )
+                      Spacer(modifier = Modifier.height(4.dp))
+                      Text(
+                        text = when (quickLink) {
+                          is HelpCenterUiState.QuickLinkType.CommonClaimType -> {
+                            val hintTextRes = quickLink.commonClaim.hintTextRes
+                            if (hintTextRes != null) stringResource(hintTextRes) else ""
+                          }
+                          is HelpCenterUiState.QuickLinkType.QuickActionType -> stringResource(
+                            quickLink.quickAction.hintTextRes,
+                          )
+                        },
+                        textAlign = TextAlign.Start,
+                        modifier = Modifier.padding(horizontal = 16.dp),
+                        color = MaterialTheme.colorScheme.onSurfaceVariant,
+                        fontSize = MaterialTheme.typography.titleSmall.fontSize,
```

Comment:
I'd suggest you just pass the style completely here instead of only changing the font size.
Something like `style = MaterialTheme.typography.titleSmall`